src/components/activities/ActivityComments/AddComment.tsx

```diff
@@ -108,20 +120,42 @@ export const AddComment = ({ activityId, activityType, activityPhase }: AddComme
               </Link>
             ) : (
               <Link href={`/reagir-a-une-activite/1?responseActivityId=${activityId}&responseActivityType=${activityType}`} passHref>
```

Comment:
You need to remove the <Link> if isObservator. The condition will only show the tooltip, but if you click the button, you will still be redirected to the index page to react to an activity.